app/API/V1/Controllers/APIControllerAbstract.php

```diff
@@ -44,23 +44,24 @@ public function __construct()
      */
 	public function getUser():?HasIdContract
 	{
-		try
-		{
-			if(($user = JWTAuth::parseToken()->authenticate()) === FALSE)
+        $user = null;
```

Comment:
@jerazo quick question, while I think that returning a null user is the right thing to do here, do we have a way to make a user required endpoint? So that if it turns null then we get an error? If not can you make a ticket to add a way to handle that?

Reply:
I guess any where teh acl is applied will error if no user so we should be safe